x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/parser/StatementParserTests.java

```diff
@@ -338,17 +338,17 @@ public void testInlineStatsWithoutGroups() {
     }
 
     public void testIdentifiersAsIndexPattern() {
-        assertIdentifierAsIndexPattern("foo", "from `foo`");
-        assertIdentifierAsIndexPattern("foo,test-*", "from `foo`,`test-*`");
+        // assertIdentifierAsIndexPattern("foo", "from `foo`");
+        // assertIdentifierAsIndexPattern("foo,test-*", "from `foo`,`test-*`");
```

Comment:
It's a bit awkward to have assertions that are commented out, is this a leftover?